### DISCUSSION TITLE
No Camera signal

### DISCUSSION THREAD

**Discussion by Dafics** [iNavFlight/inav#7274] [Q&A]
I am setting up a Matek F405-WSE and I am not getting a video broadcast.
I have a HGLRC 800mw VTX which acknowledged with the STATUS command as present. I have set the goggles to the Band/Channel indicated (also changed to try different Band/Channel).
I have checked the camera is connected to GND and receiving 8V. No voltage on the "CAM" pin.
VTX UART set to IRC TRAMP as per specs. CAM UART tried disabled, RunCam device.
The goggles show a blank screen, not snow, on the selected Band/Channel so I am assuming the VTX is broadcasting.
Tried a different camera (3 wire only) and no success.

What am I missing?

[diff.txt](https://github.com/iNavFlight/inav/files/6827172/diff.txt)
[status.txt](https://github.com/iNavFlight/inav/files/6827174/status.txt)

Thanks
John

**Comment by BryanFPV** [comment]
I ran into the same issue on my DALRC F405 FC, and I'm using the same VTX as yours. One more thing I did is to connect the cam video out directly to the VTX vin, everything worked perfectly fine, I can see live video in my goggle. Also, the same config worked fine in Betaflight, so I'm pretty sure something is wrong in iNAV, don't know how to fix though...

**Comment by Dafics** [comment]
Problem solved!
A pin on the VTX socket had made it’s own home in the opposing side. 
Look at pin side and it appeared all good, 6 pins straight although one slightly bent. Look at the socket side, 7 holes! The bent pin had created it’s own home.
Discard the plug/socket and solder direct and then fully operational.
